build.gradle

```diff
@@ -70,3 +70,7 @@ shadowJar {
 }
 
 defaultTasks 'clean', 'test'
+
+run {
```

Comment:
Nice job identifying this in the build.gradle file. Adding this will increase the defensiveness of our code!